src/GlobalStyles.js

```diff
@@ -3,27 +3,46 @@ import { createGlobalStyle } from "styled-components";
 const GlobalStyle = createGlobalStyle`
 * {
     box-sizing: border-box;
+    margin:0;
 }
 body {
   margin: 0;
-  font-family: sans-serif;
+  font-family: Helvetica Neue;
   font-size: 112.5%;
   background-color: white;
+  letter-spacing: 0.1em;
 ;
 }
 input,
 button,
-textarea {
+textarea, 
+p {
   font-size: inherit;
+  font-family: Helvetica Neue;
+  font-weight: 200;
+}
+
+h1{
+  display: grid;
+  justify-content: center;
+  color: var(--black);
+  font-size: 20px;
+  text-transform: uppercase;
+  font-family: Helvetica Neue;
+  letter-spacing: 0.1em;
+  font-weight: 200;
 }
 
 :root{
-  --blue: #1C85E8;
-  --peach: #FF896D;
-  --light-peach: #1C85E8;
-  --light-blue: #ACD3F8;
+  --purple: #A0568D;
+  --light-grey: #A09B9B;
+  --light-purple: #ECCDDF;
   --white: #FFFFFF;
-  --black: #212121;
+  --black: #393636;
+  --font-family: Helvetica Neue;
+  --text-transform: uppercase;
```

Comment:
I'd add the `:root` styles right after the universal selector so you can use them right away. Also, do you need the `--font-family` variable when you set it in the body?

Reply:
right! I decided to put it only in the root style now. Or do you think it suits better in the body part instead?

Reply:
No, I think that is fine. 👍

Reply:
Okay, the question regarding the right part for the font-family in GlobalStyles is answered. If I don´t put it into the body, there´s the wrong font on the searchpage.